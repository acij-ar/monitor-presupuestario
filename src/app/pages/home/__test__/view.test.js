const React = require('react');
const { shallow } = require('enzyme');

const Home = require('../view');

describe('Home page', () => {
  it('should match snapshot', () => {
    const props = {
      pageName: 'home',
    }
    const wrapper = shallow(<Home {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
