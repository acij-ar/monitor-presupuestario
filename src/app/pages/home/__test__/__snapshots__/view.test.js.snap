// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should match snapshot 1`] = `
<Page
  pageName="home"
>
  <a
    href="/monitor"
    id="home-link"
  >
    <img
      alt="Monitor presupuestario"
      id="background-768"
      src="/static/portada-768.jpg"
    />
    <div
      id="background-1920"
    />
  </a>
</Page>
`;
